source/blog/2018-07-13-the-ember-times-issue-55.md

```diff
@@ -21,7 +21,10 @@ Check out what's been going in Emberland 🐹 these days:
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Don't Worry, Ember CLI Got You Covered](https://github.com/ember-cli/ember-cli-update)
+The **number one** tool for updating Ember.js apps or addons just got **even better**. The [newest version](https://github.com/ember-cli/ember-cli-update/releases) of `ember-cli-update` now runs `qunit-dom-codemod` for you. This mean that you with close to no effort at all can utilize this great addon for your tests.
```

Comment:
I would add some commas:
This mean that you, with close to no effort at all, can utilize this great addon for your tests.